### ISSUE TITLE
[web] noIcon and onClear on searchBar gives a warning:  React does not recognize the `noIcon` prop on a DOM element.

### ISSUE THREAD

**Issue by zivchen** [react-native-training/react-native-elements#1878]
**Explain what you did**
SearchBar with noIcon prop compiled with react-native-web

**Expected behaviour**
No warning

**Describe the bug**
noIcon:
> 
> Warning: React does not recognize the `noIcon` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `noicon` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
>     in input (created by TextInput)
>     in TextInput (at Input.js:111)

onClear:

> react-dom.development.js?61bb:506 Warning: Unknown event handler property `onClear`. It will be ignored.
>     in input (created by TextInput)
>     in TextInput (at Input.js:111)
>     in div (created by View)
>     in View (created by AnimatedComponent)
>     in AnimatedComponent (at Input.js:93)

**To Reproduce**
SearchBar with noIcon prop compiled with react-native-web

**Possible Solution**
Dont pass noIcon and onClear prop to TextInput

**Your Environment (please complete the following information):**
| software              | version |
| --------------------- | ------- |
| react-native-elements |  1.1.0       |
| react-native          |  0.58.6       |
| npm or yarn           |         |

**Comment by iRoachie** [comment]
The `noIcon` prop doesn't exist anymore after v1. Instead just set the `searchIcon={null}`

**Comment by ceribbo** [comment]
Actually I'm having the same problem, any way to solve it?

**Comment by iRoachie** [comment]
Closing as inactive. If the above solution didn't work, feel free to message or create a new issue if this is still a concern.

**Comment by ceribbo** [comment]
Actually the problem is still happening when just showing the 
`<SearchBar>`
with no params, i get the precedent cited error:
````
Unknown event handler property `onClear`. It will be ignored.
    in input (created by TextInput)
    in TextInput (at Input.js:111)
    in div (created by View)...
````
and not icon is not showed.
Any glue how to solve the problem?
I have the following react-native versions
"react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
"react-native-elements": "^1.1.0",

**Comment by zivchen** [comment]
Same here, still happening
"react-native-web": "^0.11.4",
"react-native-elements": "^1.1.0",
"react-native": "0.59.8",

Error:
Warning: Unknown event handler property `onClear`. It will be ignored.
    in input (created by TextInput)
    in TextInput (at Input.js:111)

I think it happens because of you pass rest and {...attributes} to input on SearchBar-default.js

**Comment by webnah** [comment]
> attributes

yes definitely line 91: I think it happens because of you pass rest and {...attributes} to input on SearchBar-default.js

it should be

error gone!!! cheer closed react-native-web 0.11.7
